server/modules/user/repository/userAnalysisRepository.js

```diff
@@ -1,15 +1,33 @@
 import { db } from '@server/db/db'
 
```

Comment:
and this repository to analysis module as well?

Reply:
yes, of course